RandomStudio/random-studio
# TARGET MR: Add Video focus mode

- Adds video focus mode page/route
- Rewrites BunnyCDN logic to be accessible from client/server
- Renames feature flag to hasFocusMode


**Steps to test**
1.  Give the site's videos a quick regression test. Doesn't need to be every video. Check a few case studies, home, projects, etc. Checking that they still load video data in general.
2. Verify that the two new video focus mode routes get data:

/video/daad8832-bdb5-418a-9c02-8deacf0baa9b
/video/daad8832-bdb5-418a-9c02-8deacf0baa9b/project
